docs/go/README.md

```diff
@@ -241,10 +241,10 @@ $ docker run -it --rm --env PORT=5006 -p 5006:5006 my-image-name
  - using code:	gin.SetMode(gin.ReleaseMode)
 
 [GIN-debug] Loaded HTML Templates (4): 
+	- 
 	- header.tmpl.html
 	- index.tmpl.html
 	- nav.tmpl.html
-	- 
```

Comment:
@joshwlewis The Go CNB's output here seems to vary from run to run. Is this expected?